host/OEHOSTMR/README.md

```diff
@@ -0,0 +1,3 @@
+The `liboehostmr` is a subset of liboehost, and it's main functionality is to get the enclave's measurement. Following other host libraries' naming convention, this library starts with liboehost, and `mr` is short for `measurement report`.
+
+To reuse source code, a macro `OEHOSTMR` is used that some sentences refering to libsgx_enclave_common are stripped off. Thus this library has no dependency on libsgx_enclave_common(also libsgx_dcap_ql).
```

Comment:
grammar: can't parse "is used that some sentences … are stripped off".
Please reword.   Also add space before open paren.

Reply:
Fixed, thanks.